maintainers/maintainer-list.nix

```
@@ -7582,6 +7582,12 @@
     githubId = 26231126;
     name = "Nils ANDRÉ-CHANG";
   };
+  nils-degroot = {
+    email = "nils@peeko.nl";
+    github = "nils-degroot";
+    githubId = ;
```

```suggestion
    githubId = 53556985;
```

per https://api.github.com/users/nils-degroot